Model/Config.php

```diff
@@ -347,7 +347,7 @@ public function getConfigFromDb(string $path, string $scope = ScopeInterface::SC
      *
      * @param bool $withDefault
      * @param bool $onlyActive
-     * @return array<mixed>
+     * @return array <mixed>
```

Comment:
let's revert this, php stan doesn't works with spaces:
![image](https://user-images.githubusercontent.com/10895971/160876399-91704fec-cc9e-4a75-9699-324a7e18abb4.png) 

![image](https://user-images.githubusercontent.com/10895971/160876424-372fde1c-10e0-403d-a541-e39ec42ff5a0.png)